var/spack/repos/builtin/packages/atop/package.py

```diff
@@ -28,10 +28,13 @@
 class Atop(Package):
     """Atop is an ASCII full-screen performance monitor for Linux"""
     homepage = "http://www.atoptool.nl/index.php"
-    url      = "http://www.atoptool.nl/download/atop-2.2-3.tar.gz"
+    url = "http://www.atoptool.nl/download/atop-2.2-3.tar.gz"
```

Comment:
Yes, but I got flake8 it through ```yum install python-flake8``` instead of pip. That version is very old now that I look at it.